bindings/go/scip/internal/old_symbol_parser.go

```diff
@@ -0,0 +1,239 @@
+package internal
```

Comment:
We can remove this code after the new symbol parser doesn't show any problems in practice.